test/cursor.cpp

```diff
@@ -108,3 +111,32 @@ TEST_CASE("values, scoped watching")
     commit(st);
     CHECK(2 == s.count());
 }
+
+struct yearday
+{
+    int day;
+    int month;
+};
+BOOST_FUSION_ADAPT_STRUCT(yearday, day, month);
+
+struct person
+{
+    yearday birthday;
+    std::string name;
+    std::vector<std::string> things {};
+};
+BOOST_FUSION_ADAPT_STRUCT(person, birthday, name);
+
+using boost::fusion::operators::operator==;
+using boost::fusion::operators::operator!=;
+
+TEST_CASE("zooming, unfocused")
+{
+    state<std::vector<person>> st{{person{{5, 4}, "juanpe"}}};
+    auto p  = st[0];
+    auto n1 = p[&person::name];
+//    auto n2 = p[lens::attr(&person::name)];
+//    reader<std::optional<std::string>> n2 = p[lens::attr(&person::name)];
```

Comment:
What is this commented stuff about?

Reply:
this is the syntax I was trying to SFINAE on, but couldn't because of the return type deduction shenanigans.
these lines won't compile, which is better than nothing at least.